src/lambda-edge/parse-auth/index.ts

```diff
@@ -34,7 +34,14 @@ export const handler: CloudFrontRequestHandler = async (event) => {
             code_verifier: pkce
         });
 
-        const res = await httpPostWithRetry(`https://${cognitoAuthDomain}/oauth2/token`, body, { headers: { 'Content-Type': 'application/x-www-form-urlencoded' } });
+        let headers: { 'Content-Type': string, Authorization?: string } = { 'Content-Type': 'application/x-www-form-urlencoded' }
```

Comment:
Can be `const` right? Not reassigning to that variable.

Reply:
Updated